sdk/relay/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
remove this file as we don't need to preview test env

Reply:
Removed